index.html

```diff
@@ -289,6 +289,28 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <footer class="contact_form">
+        <h2 class="contact_header">
+          I'm always interested in hearing
+           about new projects, 
+          so if you'd like to
+           chat please get in touch.
+        </h2>
+        <form action="#" method="post" class="form">
```

Comment:
- [ ] When I click enter after filling the form my data is not saved in formspree. It is important to save users data in form spree for future purpose

Reply:
Thank you, I will Implement this suggestion.